www/install/php/Update-20.04.0-beta.1.php

```diff
@@ -135,17 +136,31 @@
 
     // checking if the instance is a central and generating the configuration file
     if ($isACentral === true) {
-        $errorMessage = 'Gorgone template file is missing';
-        if (!file_exists(__DIR__ . '/../var/configFileGorgoneTemplate')) {
+        if (!file_exists(__DIR__ . '/../var/databaseTemplate.yaml')) {
```

Comment:
to stay consistent with other file checks, should we add check of filesize ?

Reply:
indeed I'll add this check